go/vt/mysqlctl/compression_test.go

```diff
@@ -194,3 +207,29 @@ func TestValidateExternalCmd(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateCompressionEngineName(t *testing.T) {
```

Comment:
How is this test different from `TestUnsupprtedBuiltinCompressors`?

Reply:
mainly builtin will fail 'external' but validcompressionEngineName will not fail 'external'